modules/cron/setting.go

```diff
@@ -40,6 +40,14 @@ type UpdateExistingConfig struct {
 	UpdateExisting bool
 }
 
+// CleanupHookTaskConfig represents a cron task with settings to cleanup hook_task
+type CleanupHookTaskConfig struct {
+	BaseConfig
+	CleanupType  string
+	OlderThan    time.Duration
+	NumberToKeep int
+}
+
```

Comment:
As this is not used anywhere else I'd say that this could just go in to the registerCleanupHookTaskTable function (similar to the RepoHealthCheckConfig struct in registerRepoHealthCheck.)